data/remnant/remnant 2 side missions.txt

```diff
@@ -104,7 +104,7 @@ mission "Remnant: Visit to Paradise 1"
 				`	"Thanks for the explanation."`
 					goto ourdestination
 				`	"What is a coefficient?"`
-			`	"Here, I will switch to base 10 to make it easier for you to understand." She pulls up a hologram with 299,792,458 m/s on it. As she speaks, the number shifts to 2.99792458x10^8 m/s. "When we write out a number such as the speed of light, it is typical to break it into parts. This is generally referred to as scientific notation. This is done to facilitate comparing and working with large numbers. The first part, 2.99792458 in this case, would be called the coefficient. The second part, x10, specifies the base; and the third part, ^8 is the exponent." The display shifts again to show 1.1DE784Ax16^7 m/s. "So this would be how we typically write the speed of light when we are using it in our research. So, 1.1DE78 is the coefficient of both the speed of light and the speed of the train, just with a different exponent." She pauses, then vanishes the display. "We will be arriving momentarily, so I should probably stop there."`
+			`	"Here, I will switch to base 10 to make it easier for you to understand." She pulls up a hologram with 299,792,458 m/s on it. As she speaks, the number shifts to 2.99792458x10^8 m/s. "When we write out a number such as the speed of light, it is typical to break it into parts. This is generally referred to as scientific notation. This is done to facilitate comparing and working with large numbers. The first part, 2.99792458 in this case, would be called the coefficient. The second part, x10, specifies the base; and the third part, ^8 is the exponent." The display shifts again to show 1.1DE784Ax10^7 m/s. "So this would be how we typically write the speed of light when we are using it in our research. So, 1.1DE78 is the coefficient of both the speed of light and the speed of the train, just with a different exponent." She pauses, then vanishes the display. "We will be arriving momentarily, so I should probably stop there."`
```

Comment:
As @warp-core noted, the line "The second part, x10, specifies the base" is not true, or at least is weirdly worded.

Reply:
![Scientific-notation](https://user-images.githubusercontent.com/32169904/213887789-369f2d57-ebe2-472e-b0a4-67568a6dcc44.png)

The "10" is properly referred to as a base, just to be clear. 

It *is* a rather confusing situation, though, since "base" can refer to two different things.

Reply:
It may be called the *base*, but to say that it specifies anything other than the boundary between the coefficient and exponent is, at best, misleading if it's just going to be written in the same base as all the other digits.